virttest/utils_test/libvirt.py

```diff
@@ -3505,11 +3505,13 @@ def modify_vm_iface(vm_name, oper, iface_dict, index=0, virsh_instance=virsh):
     iface_virtualport_type = iface_dict.get('virtualport_type')
     iface_filter_parameters = iface_dict.get('filter_parameters')
     iface_port = iface_dict.get('port')
+    iface_coalesce = iface_dict.get('coalesce')
     del_addr = iface_dict.get('del_addr')
     del_rom = iface_dict.get('del_rom')
     del_filter = iface_dict.get('del_filter')
     del_port = iface_dict.get('del_port')
-    del_mac = "yes" == iface_dict.get('del_mac', "no")
+    del_mac = iface_dict.get('del_mac')
```

Comment:
May I ask why we need to change this line?

Reply:
@Yingshun Yes, of course. The reason that I made this change is that I have checked current test case in iface_update and found that the old implementation is not working with current test code. So I did a little change to both sides to make sure those test cases works. I guess it's a kind of fix.

Reply:
Ok, got it. Thanks for your explanation!

Reply:
@kylazhang you're right! I'll update nwfilter_vm_start.